test/integration/smoke/test_volumes.py

```diff
@@ -814,7 +814,7 @@ def test_10_list_volumes(self):
         host = Host.list(
             self.apiclient,
             type='Routing',
-            virtualmachineid=list_vm.id
```

Comment:
LGTM. While this should have worked, listHosts when passed a vm id will also return other hosts in the cluster that can host this VM (enough VM cpu/ram)